capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_test.go

```diff
@@ -460,87 +345,89 @@ func Test_formatMessage(t *testing.T) {
 func Test_RunIncludedFieldsAndExtensionsChecks(t *testing.T) {
 	setupCapabilityStatement(t, filepath.Join("../../testdata", "cerner_capability_dstu2.json"))
 	capInt := testQueueMsg["capabilityStatement"].(map[string]interface{})
-	includedFields := RunIncludedFieldsAndExtensionsChecks(capInt)
+	fhirVersion := "1.0.2"
+	includedFields := RunIncludedFieldsAndExtensionsChecks(capInt, fhirVersion)
 	th.Assert(t, includedFields[0].Exists == true, "Expected url in includedFields to be true, was false")
 	th.Assert(t, includedFields[2].Exists == true, "Expected name in includedFields to be true, was false")
-	th.Assert(t, includedFields[8].Exists == false, "Expected contact in includedFields to be false, was true")
-	th.Assert(t, includedFields[18].Exists == false, "Expected software.name in includedFields to be false, was true")
-	th.Assert(t, includedFields[25].Exists == true, "Expected format in includedFields to be true, was false")
-	th.Assert(t, includedFields[32].Exists == true, "Expected rest.mode in includedFields to be true, was false")
-	th.Assert(t, includedFields[37].Exists == false, "Expected rest.resource.conditionalDelete in includedFields to be false, was true")
-	th.Assert(t, includedFields[39].Exists == true, "Expected rest.resource.searchParam.type in includedFields to be true, was false")
-	th.Assert(t, includedFields[55].Exists == false, "Expected conformance expectation extension in includedFields to be false, was true")
-	th.Assert(t, includedFields[50].Exists == true, "Expected oauth-uris extension in includedFields to be true, was false")
-	th.Assert(t, includedFields[46].Exists == false, "Expected capabilities extension in includedFields to be false, was true")
+	th.Assert(t, includedFields[6].Exists == false, "Expected contact in includedFields to be false, was true")
+	th.Assert(t, includedFields[11].Exists == false, "Expected software.name in includedFields to be false, was true")
+	th.Assert(t, includedFields[17].Exists == true, "Expected format in includedFields to be true, was false")
+	th.Assert(t, includedFields[18].Exists == true, "Expected rest.mode in includedFields to be true, was false")
+	th.Assert(t, includedFields[22].Exists == false, "Expected rest.resource.conditionalDelete in includedFields to be false, was true")
+	th.Assert(t, includedFields[23].Exists == true, "Expected rest.resource.searchParam.type in includedFields to be true, was false")
+	th.Assert(t, includedFields[33].Exists == false, "Expected conformance expectation extension in includedFields to be false, was true")
+	th.Assert(t, includedFields[35].Exists == true, "Expected DSTU2oauth-uris1 extension in includedFields to be true, was false")
```

Comment:
Should this be DSTU2-oauth-uris?

Reply:
Yep, this must have been a typo!

Reply:
Yep that must have been a typo, good catch!